tests/Fixtures/Integration/misc/phpdoc_to_comment,phpdoc_var_without_name.test

```diff
@@ -8,19 +8,35 @@ Integration of fixers: phpdoc_to_comment,phpdoc_var_without_name.
 --EXPECT--
```

Comment:
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3911#discussion_r202366775
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3911/files#r208303314

basically, this test got rewritten.

I just tested locally original test against code changes proposed by this PR - the test is failing. This means that the rule behind the fixer got modified, and test that has to be changed (regardless if it's utest or itest) proves that.
I find this to be a BC breaker - if a fixer apply single rule in multiple places (like here - for properties and for variables), and you want to adjust the scope and allow to fix only some of places - it deserves configurable option. If you want to fix only some of places without being able to configure it, it's a BC breaker for the rule.

Now, if I would merge it, nobody would be able to apply the rule for variables.

Reply:
@julienfalque can you weigh in here? Wasn't this the entire point of your issue?

Reply:
if someone wants to apply the rule only for properties, i'm fine with that.
but allowing to do so must not disallow to apply the rule for variables, as this is the behaviour we currently have - thus changing the rule in a way that variables will not be modified anymore is BC breaker.

I agree - i am not fully aware about the concerns or issues and i'm far from judging should we fix only properties or everyone - I'm only watching after keeping BC.

Reply:
From [phpDocumentator doc](https://docs.phpdoc.org/references/phpdoc/tags/var.html) and [PSR-19 PHPDoc tags](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#description-16):
> The `@var` tag MUST contain the name of the element it documents. An exception to this is when property declarations only refer to a single property. In this case the name of the property MAY be omitted.

IMO the current behavior of the fixer is wrong, I consider changing it a bugfix and I'm fine with breaking BC here.

Reply:
If the fixer would be:
> `@var` tag must follow 5.17 of PSR-19 proposal

I would agree.

Yet, it's different:
```
ker@dus:~/github/PHP-CS-Fixer λ php php-cs-fixer describe phpdoc_var_without_name
Description of phpdoc_var_without_name rule.
`@var` and `@type` annotations should not contain the variable name.
```

then, i would suggest to:
- make one fixer like `phpdoc_property_var_without_name` (this PR, just new fixer instead of modifying existing)
- deprecate existing fixer
- add fixer like `phpdoc_variable_var_with_name` (our current, existing fixer does the oposite) - of course, a separated rule

Reply:
I considered the old working of the fixer a bug as well, together with other members and as such merged this.
I've updated the description to be more precise on the working of the fixer. If anyone needs the `phpdoc_variable_var_with_name` fixer than a PR is welcome.